src/main/java/com/sap/cloudfoundry/client/facade/domain/Staging.java

```diff
@@ -61,7 +61,10 @@
      * @return the stack to use when staging the application, or null to use the default stack
      */
     @Nullable
-    String getStack();
+    String getStackName();
+
+    @Nullable
+    UUID getStackId();
```

Comment:
Why is this needed here?

Reply:
Because it's the only way to retrieve the id of a stack in order to use it as a key for the StackCache. At least the only logical way since we retrieve the stack's name from here.